R/sem.R

```diff
@@ -167,7 +167,7 @@ checkLavaanModel <- function(model, availableVars) {
 
   # Check variable names
   if (!missing(availableVars)) {
-    latents <- unique(parsed[parsed$op == "=~",]$lhs)
+    latents <- unique(parsed[parsed$op == "=~"]$lhs)
```

Comment:
`parsed` is a data.frame, right? Doesn't this change break SEM then? 
```suggestion
    latents <- unique(parsed[parsed$op == "=~",]$lhs)
```